__tests__/app.test.js

```diff
@@ -0,0 +1,21 @@
+const request = require('supertest');
+const app = require('../server/app.js');
+
+describe('Test the root path', () => {
+  test('It should respond to GET method', () => {
+    return request(app)
+      .get('/')
+      .then(response => {
+        expect(response.statusCode).toBe(200)
+      })
+  });
+
+  test('It should respond to GET method', () => {
```

Comment:
For this part to not fail, I believe you will have to mock the Person.js file's database query or setup the database in circleCI accordingly